test/test_pipeline/test_tabular_classification.py

```diff
@@ -53,7 +52,6 @@ def _assert_pipeline_search_space(self, pipeline, search_space_updates):
             elif isinstance(hyperparameter, CategoricalHyperparameter):
                 assert update.value_range == hyperparameter.choices
 
-    @flaky.flaky(max_runs=2)
```

Comment:
We are sampling a random configuration, wouldn't it make it fail?

Reply:
for the fit, predict functions, I dont think it should fail. I'll add it for the score function if that's what you mean

Reply:
I just checked that the score function is using default config

Reply:
@ravinkohli You mean the result for this test is deterministic and we do not need to repeat anymore?
If it is a stochastic test, it is better to keep it for more success rate. (flaky does not do anything bad if we pass the test by ,let's say, 99%, right?)

Reply:
if you're talking about the fit and predict functions, I'd say they are quite deterministic.